### DISCUSSION TITLE
Numbers on the y axis are too close to each other if the 'valueScale' is set to symlog, and unable to show the lables on top of bar in grouped mode. #1613

### DISCUSSION THREAD

**Discussion by vikrantsingh47** [plouc/nivo#1660] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/75779649/122947888-1f62f200-d398-11eb-8766-f5dbeefa73da.png)
as you can see from the above screenshot, the numbers on the y-axis are too close to each other if 'valueScale' is set to symlog instead of 'linear'.
Any way to remove this issue?
 my second issue is that, is it possible to display the labels on top of the bars instead of in the middle of the bar in grouped mode?
![image](https://user-images.githubusercontent.com/4192343/124472094-5505d300-ddbb-11eb-8772-30c49880f1d2.png)

**Comment by wyze** [comment]
Instead of screenshots of output, please provide CodeSandboxes as they are easier to work with and we know the code you are using.

**Comment by vikrantsingh47** [comment]
@wyze sorry for being late, here is the sandbox. if the `valueScale ` is `symlog` the y-axis numbers are too close to each other. 
Also, i want to show the labels on top of the bars instead of in the middle in `grouped` mode
https://codesandbox.io/s/react-nivo-valuescale-5o9wu?file=/src/App.js

**Comment by wyze** [comment, marked as answer]
You can limit the ticks by using `axisLeft.tickValues` to only get a certain amount. If you want to keep that amount there is a `constant` prop you can pass to the scale type, but that will also shrink your bars.

The labels, you should disable the default labels and render them in a custom layer using the bar information to position them.

https://codesandbox.io/s/react-nivo-valuescale-forked-lb954?file=/src/App.js
